desktop/tests/test_persona.py

```
@@ -0,0 +1,40 @@
+    buddy_prompt = system_prompt(store, buddy)
+    sourcing_prompt = system_prompt(store, sourcing)
+    assert buddy.body in buddy_prompt
+    assert "Sourcecado's sourcing agent" not in buddy_prompt
+    assert sourcing.body in sourcing_prompt
+    assert buddy.body not in sourcing_prompt
+
+
+def test_parse_rejects_missing_frontmatter():
+    try:
+        parse_persona("just a body")
+        assert False, "expected ManifestError"
```

Ruff (0.16.2)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)